plugins/chain/serialize_state.cpp

```diff
@@ -297,6 +301,17 @@ void plugin::serialize_state(const bfs::path& output) {
         wlog("Data SHA256 hash: ${h}", ("h", out.hash().str()));
         out.close();
 
+        auto rep_file = output;
+        rep_file += ".reputation";
+        if (db_.has_index<golos::plugins::follow::reputation_index>()) {
```

Comment:
prefer to have option so user can express intention to have reputation index stored.
but ok in this case.

Reply:
user can remove this file before passing to create-genesis tool
same approach with operation dump - some of files can be deleted before passing to tool

Reply:
ok